modules/ROOT/pages/schemas.adoc

```diff
@@ -283,6 +283,51 @@ numberOfSIMCards: int
 
 |===
 
+== Mutations and Input Types
+
+Mutations are operations you use to add, update and delete data objects. In GraphQL, mutation operations represent the POST, PUT, DELETE, and HTTP methods you’ve defined in your REST API specification. 
+
+For example, this RAML specification defines a POST endpoint at `/createOrder/\{OrderInput}`, the mutation method in the schema is `createOrder(input:OrderInput)`:
+
+[%header,%autowidth.spread,cols="a,a"]
+|===
+| RAML API Specification | API Schema
+|
+[source]
+--
+#%RAML 1.0
+title: Order API
+
+/createOrder/{OrderInput}:
+  post:
+    displayName: Create Order by OrderInput
+--
+
+|
+[source]
+--
+createOrder(input:OrderInput): Order
+--
+
+|===
+
+Input types represent the attributes of objects created by mutation operations. They are POST, PUT, DELETE, and request objects you use to modify information in your unified schema when you run mutations.
```

Comment:
This is likely understood by your audience, but not so much to me, so...due diligence:

- Here and earlier you use the term "represent," but I don't know if you mean that A is B (types are attributes) or A is symbolic of B (types are equivalent to attributes). This may be me overthinking, but maybe search for "represent" and evaluate for precision.
- To what does "They" refer: input types, attributes of objects, or mutation operations?
- Does "They" = four things (POST, PUT, DELETE, and objects) or three things (POST, PUT, and DELETE) that request objects? (Is "request" an adjective modifying "objects" or a verb?)